src/controllers/controller.line.js

```diff
@@ -73,8 +73,8 @@ module.exports = DatasetController.extend({
 		var line = meta.dataset;
 		var points = meta.data || [];
 		var options = me.chart.options;
-		var dataset = me.getDataset();
-		var showLine = me._showLine = valueOrDefault(me._config.showLine, options.showLines);
+		var datasetOpts = me._config;
```

Comment:
What about starting to call the alias of `this._config` simply `var config = me._config`?

Reply:
That makes sense as `_config` is controller's internal config values at this point. Done.